cookbooks/hdfsdu/attributes/default.rb

```diff
@@ -0,0 +1,47 @@
+##################################
+#  hdfsdu specific attributes  #
+##################################
+
+default[:hdfsdu][:repo_url] = 'https://github.com/twitter/hdfs-du.git'
+default[:hdfsdu][:repo_branch] = 'master'
+default[:hdfsdu][:version] = '0.1.0'
+
+default[:hdfsdu][:download_dir] = '/tmp'
+default[:hdfsdu][:install_dir] = '/usr/lib'
+default[:hdfsdu][:service_dir] = '/etc/init'
+default[:hdfsdu][:log_dir] = '/var/log/hdfsdu'
+default[:hdfsdu][:data_dir] = '/var/lib/hdfsdu/data'
+default[:hdfsdu][:bin_dir] = '/home/vagrant/chef-bcpc/bins'
```

Comment:
For a generic cookbook this seems like a high-bar; perhaps use a HOME env. var. or equivalent and a `bcpc-hadoop` wrapper could set this for folks using the BACH stack?